index.html

```diff
@@ -7,37 +7,87 @@
    
   </head>
   <body>
+    <header>
+      <nav>
+        <ol>
+          <li id="AboutLink"><a href="#About">About</a></li> 
+          <li id="ExperienceLink"><a href="#Experience">Experience</a></li>
+          <li id="SkillsLink"><a href="#skills"> Skills</a></li>
+          <li id="ProjectsLink"><a href="#projects"> Projects</a></li>
+          <li id="ConnectLink"><a href="#connect"> connect</a></li>
```

Comment:
It looks like your navigation links are in a different order than the sections. For example, Projects is the last item on the page, but it's the 4rd item on your navigation bar.

I'd also recommend using similar conventions for your sections names. For example, `About` is capitalized, but `skills` is not. It works just fine, but is cleaner to choose one or the other.